pages/contact.js

```diff
@@ -6,67 +7,98 @@ import Row from 'react-bootstrap/Row';
-            />
+const host = 'http://localhost';
+const port = 3001;
 
-            <NavbarBootstrap isAuthenticated={false} />
+class Contact extends Component {
+    constructor(props) {
+        super(props);
 
-            <Container className="my-3">
-                <Row className="justify-content-center">
-                    <Col md={6}>
-                        <Card>
-                            <Card.Body>
-                                <Card.Title>Contact Us</Card.Title>
-                                <Card.Text>
-                                    <Form action="https://formspree.io/brakessupreme@gmail.com" method="POST" >
-                                        <Form.Group controlId="nameInput">
-                                            <Form.Label>Name</Form.Label>
-                                            <Form.Control
-                                                name="name"
-                                                type="text"
-                                                placeholder="John Smith"
-                                                required
-                                            />
-                                        </Form.Group>
-                                        <Form.Group controlId="emailInput">
-                                            <Form.Label>Email address</Form.Label>
-                                            <Form.Control
-                                                name="email"
-                                                type="email"
-                                                placeholder="john.smith@mail.me"
-                                                required
-                                            />
-                                        </Form.Group>
-                                        <Form.Group controlId="messageTextarea">
-                                            <Form.Label>Message</Form.Label>
-                                            <Form.Control
-                                                name="message"
-                                                as="textarea"
-                                                rows="5"
-                                                placeholder="Go ahead, we're listening..."
-                                                required
-                                            />
-                                        </Form.Group>
-                                        <Button variant="outline-info" type="submit">
-                                            Submit
-                                        </Button>
-                                    </Form>
-                                </Card.Text>
-                            </Card.Body>
-                        </Card>
-                    </Col>
-                </Row>
-            </Container>
-        </div>
-    );
+        this.state = {
+            isAuthenticated: false,
```

Comment:
How will the usage of the contact page differ now that we know if the user is logged in or not? I am assuming, this doesn't change anything. This state is here for UI/UX consistency?

Reply:
see comment below @arshiannafi